src/components/EnterpriseApp/EnterpriseApp.test.jsx

```diff
@@ -11,10 +12,13 @@ import { EnterpriseSubsidiesContext } from '../EnterpriseSubsidiesContext';
 
 features.SETTINGS_PAGE = true;
 
+// eslint-disable-next-line react/function-component-definition
 const EnterpriseSubsidiesContextProvider = ({ children }) => (
-  <EnterpriseSubsidiesContext.Provider value={{
-    canManageLearnerCredit: true,
-  }}
+  <EnterpriseSubsidiesContext.Provider value={
+    useMemo(() => ({
+      canManageLearnerCredit: true,
+    }), [])
```

Comment:
Don't call `useMemo` inside of the render function... move it above the render function and save it as a new variable to be used here instead.